mavencentral-properties.gradle

```diff
@@ -0,0 +1,25 @@
+// Find property in either System environment or Gradle properties.
+// If set in both places, Gradle properties win.
+def getPropertyValue(String propertyName) {
+    def value = System.getenv(propertyName)
+    if (project.hasProperty(propertyName)) {
+        value = project.getProperty(propertyName)
+    }
+    return (value == null) ? "" : value;
```

Comment:
You can do this in one line with
```
return project.findProperty(name) ? System.getenv(name) : ""
```

Reply:
Hmm, that returns `null` if both `findProperty(...)` and `genenv(...)` doesn't find the property, which would break some logic elsewhere.

Reply:
Ah, yes. Then I guess you should just chain it with 
```
return project.findProperty(name) ?: System.getenv(name) ?: ""
```
But anyway, if the original works 🤷 . It was just point out the existence of `findProperty`.